src/components/ConditionField/ConditionField.test.js

```diff
@@ -109,3 +107,128 @@ describe('Tooltip', () => {
     ).toBeTruthy()
   })
 })
+
+describe('With selectable conjunction', () => {
+  test('should display selectable conjunction button with AND/OR options', () => {
+    render(
+      <ConditionField.Group canChangeConjunction>
+        <ConditionField />
+      </ConditionField.Group>
+    )
+
+    expect(dropdownTrigger()).toBeInTheDocument()
+
+    user.click(dropdownTrigger())
+
+    expect(conjunctionMenu()).toBeInTheDocument()
+    expect(
+      within(conjunctionMenu())
+        .getAllByRole('option')
+        .map(el => el.textContent)
+    ).toEqual(['OR', 'AND'])
+  })
+
+  test('should display OR selected when no conjunction provided', () => {
+    render(
+      <ConditionField.Group canChangeConjunction>
+        <ConditionField />
+      </ConditionField.Group>
+    )
+
+    user.click(dropdownTrigger())
+    expect(screen.getByRole('button', { name: 'or' })).toBeInTheDocument()
+    // expect(getSelectedConjunctionOption()).toHaveTextContent('OR')
+  })
+
+  test('should display AND conjunction selected', () => {
+    render(
+      <ConditionField.Group canChangeConjunction conjunction={'and'}>
+        <ConditionField />
+      </ConditionField.Group>
+    )
+
+    user.click(dropdownTrigger())
+    expect(screen.getByRole('button', { name: 'and' })).toBeInTheDocument()
+    // expect(getSelectedConjunctionOption()).toHaveTextContent('AND')
```

Comment:
Can these commented out expect calls be removed?

Reply:
Yeah, I wanted to test this, but the component doesn't support it yet. I'll remove it.